src/pages/preferences/preferencesTabs/PublicKeysTab.tsx

```diff
@@ -131,25 +25,16 @@ export const PublicKeysTab: React.FC = () => {
           });
         }}
       >
-        Add New Key
+        Add key
       </PlusButton>
       <TableContainer>
```

Comment:
you could possibly remove the TableContainer here, the space between the button and table seems like a lot